templates/adherent/list_my_administrated_citizen_projects.html.twig

```diff
@@ -1,20 +1,13 @@
-{% if citizen_projects|length > 1 %}
+{% if administrated_citizen_projects|length > 1 %}
     <div class="nav-dropdown">
         <i class="fa fa-caret-down" aria-hidden="true"></i>
         <i class="fa fa-caret-up" aria-hidden="true"></i>
         <ul class="list__links">
-            {% for citizen_project in citizen_projects|slice(0, 1) if app.user.isAdministratorOf(citizen_project) %}
-                <li>
-                    <a href="{{ path('app_citizen_project_show', { slug: citizen_project.slug }) }}"
-                       title="{{ citizen_project.nameWithDistrict }}">
-                        {{- citizen_project.nameWithDistrict -}}
-                    </a>
-                </li>
-            {% endfor %}
+            <li><a href="#">{{ 'my_citizen_projects'|trans }}</a></li>
```

Comment:
Why the `a` tag is used with a `href="#"` here ?

Reply:
It's to have a link style and the `:hover` behavior.